lib/blocks.php

```diff
@@ -132,18 +130,18 @@ function gutenberg_reregister_core_block_types() {
 			register_block_type_from_metadata( $block_json_file );
 		}
 
-		foreach ( $block_names as $file => $block_names ) {
```

Comment:
I wonder if this was creating some hidden bugs or something. reusing the same variable that we're iterating on.

Reply:
Me too :) - but I think not. Will check

Reply:
Fortunately `$block_names` is not used afterwards because the value has changed.